modules/virt-creating-vm-instancetype.adoc

```diff
@@ -6,11 +6,6 @@
 [id="virt-creating-vm-instancetype_{context}"]
 = Creating a virtual machine from an instance type
 
-[IMPORTANT]
-====
-This is a Developer Preview feature only. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
-====
-
 You can use common instance types to install and edit customized instance types and preferences to create a virtual machine (VM).
 
 .Procedure
```

Comment:
I discovered something while writing the UI doc quick reference. If I understand the guidelines correctly, we can't use field names as generic nouns. Example: `Select an *InstanceType*.` > `Select an InstanceType from the *InstanceType* list.` or `Select an InstanceType.` with no markup of `InstanceType`.

We should not mark up `VirtualMachine` with single quotes or backticks because we are not using the term as a pod or CR name. In generic contexts, it looks like we use "Create a VirtualMachine".

Reply:
Related note: Part of me is starting to wonder whether we should replace all instances of "VM" and "virtual machine" with "VirtualMachine". I might ask Kathryn Alexander or Andrea Hoffer about this.

Reply:
Re: step 1. We normally only document one way to do a task. I suggest this rewrite: `Click the *InstanceTypes* tab on the *Virtualization* -> *Catalog* page.`